standalone/runtime/src/lib.rs

```diff
@@ -1334,6 +1335,17 @@ impl pallet_mq::Config for Runtime {
     type QueueNotifyConfig = msg_routing::MessageRouteConfig;
     type CallMatcher = MqCallMatcher;
 }
+
+pub struct SetBudgetMembers;
```

Comment:
It's ok to use `EnsureSignedBy` here. An easier way is to create a storage item in the pallet to store the account, check the caller in your function by comparing the origin and the stored account, and allow the CouncilOrRoot to edit the account.

Reply:
When i check through the substrate's frame update, i found they prefer use EnsureSignedBy to check validation of callers, can see it in the latest concil pallet, we can implement a `EnsureSignedBy` trait to specify which accounts can change the concil members.